### PR TITLE
feat: add tap-instantly-ai

### PR DESCRIPTION
**Pull request by bardrink**
Adding a Meltano SDK tap for [Instantly AI](https://instantly.ai/).

### CODE REVIEW

File: _data/meltano/extractors/tap-instantly-ai/strvcom.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- schema-flattening
+- state
+- stream-maps
+description: Tap for Instantly AI
+domain_url: https://developer.instantly.ai
+executable: tap-instantly-ai
+keywords:
+- meltano_sdk
+label: Tap Instantly AI
+logo_url: /assets/logos/extractors/instantly-ai.png
```
Comment (edgarrmondragon) [inline _data/meltano/extractors/tap-instantly-ai/strvcom.yml:15]: Did you mean to add this?

**Review by edgarrmondragon — CHANGES_REQUESTED** [review]
Thanks @bardrink!

Just one comment about the missing logo.

File: _data/meltano/extractors/tap-instantly-ai/strvcom.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+maintenance_status: active
+name: tap-instantly-ai
+namespace: tap_instantly_ai
+next_steps: ''
+pip_url: git+https://github.com/strvcom/tap-instantly-ai.git
+quality: unknown
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-instantly-ai/strvcom.yml:21]:
```suggestion
quality: silver
```

File: _data/meltano/extractors/tap-instantly-ai/strvcom.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: Tap for Instantly AI
+domain_url: https://developer.instantly.ai
+executable: tap-instantly-ai
+keywords:
+- meltano_sdk
+label: Tap Instantly AI
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-instantly-ai/strvcom.yml:14]:
```suggestion
label: Instantly AI
```

File: _data/meltano/extractors/tap-instantly-ai/strvcom.yml [hunk 4]
```diff
@@ -0,0 +1,89 @@
+capabilities:
+- about
+- batch
+- catalog
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: Tap for Instantly AI
```
Comment (pnadolny13) [inline _data/meltano/extractors/tap-instantly-ai/strvcom.yml:9]:
```suggestion
description: AI-powered cold email campaign management platform
```

**Review by pnadolny13 — APPROVED** [review]

**Review by edgarrmondragon — APPROVED** [review]
Thanks and congrats on your first addition to the Hub @bardrink!

**Pull request merged**